excitebike/src/main/java/com/codenjoy/dojo/excitebike/services/Events.java

```diff
@@ -23,10 +23,13 @@
  */
 
 
-/**
- * Ивенты, которые могут возникать в игре опиши тут. Что есть ивенты? ну убили твоего героя и ты хочешь ему очков начислить штрафных
- * или, быть может, наоборот - он поднял что-то ценное и ты хочешь ему дать бонус. Вот все все ивенты.
- */
 public enum Events {
-    WIN, LOOSE;
+    ACCELERATE,
+    SLOW_DOWN,
+    JUMP,
```

Comment:
I think we dont need jump event. To be discussed.

Reply:
discussed, agreed that it's fine